build-system/test-configs/config.js

```diff
@@ -40,6 +40,12 @@ const builtRuntimePaths = [
     nocache: false,
     watched: true,
   },
+  {
```

Comment:
Since these are being fed to a glob parser (guessing `globby` but syntax is pretty standard), I think here and below you could change the existing patterns to `dist/**/*.{js,mjs}` and `dist.tools/**/*.{js,mjs}` rather than add a new rule config (we've done something similar with Owners rules)

Reply:
Agreed, the other values seem to be the same between module and nomodule files.

Reply:
done.

Reply:
<img width="1294" alt="Screen Shot 2020-09-25 at 9 57 24 AM" src="https://user-images.githubusercontent.com/354746/94295188-b8520300-ff15-11ea-92de-6904e4b6985d.png">

@rcebulko so i've been debugging the error in the screen shot yesterday and it looks like it is caused by this change. seems karma cannot handle expansion properly such as `.{js,mjs}`, reverting to my old change

Reply:
Can confirm that these globs are passed to Karma via the `files` config setting. 
https://github.com/ampproject/amphtml/blob/23deb93a250c122caa3501ad4fdd653e519c0daa/build-system/tasks/runtime-test/runtime-test-base.js#L186

The Karma [documentation](http://karma-runner.github.io/5.2/config/configuration-file.html), says it uses `minimatch` under the covers, so you may be able to do what Ryan suggested using `dist/**/*.(js|mjs)` instead of `dist/**/*.{js,mjs}`.

Reply:
@rsimha, will try it out. thanks!

Reply:
@rsimha tried using (js|mjs)  (please sanity check screenshot below), and well everything failed
<img width="805" alt="Screen Shot 2020-09-25 at 12 14 50 PM" src="https://user-images.githubusercontent.com/354746/94306985-c8bfa900-ff28-11ea-9e3f-266954f22e65.png">

Reply:
so tried it `dist/**/*.(js|mjs)` (please sanity check my screenshot below), and everything failed this time
<img width="821" alt="Screen Shot 2020-09-25 at 12 17 18 PM" src="https://user-images.githubusercontent.com/354746/94307157-176d4300-ff29-11ea-8877-bba90ac6616d.png">